src/planet-a/Active/SysPlanetA.cpp

```diff
@@ -110,11 +113,15 @@ int SysPlanetA::deactivate_sat(osp::active::ActiveScene &scene,
                                osp::universe::Satellite tgtSat,
                                osp::active::ActiveEnt tgtEnt)
 {
+    // TODO
```

Comment:
provide a one sentence summary of what the function should do when it's written.